cmd/kubelet/app/server.go

```diff
@@ -175,6 +181,12 @@ func (s *KubeletServer) Run(_ []string) error {
 		HighThresholdPercent: s.ImageGCHighThresholdPercent,
 		LowThresholdPercent:  s.ImageGCLowThresholdPercent,
 	}
+
+	cloud := cloudprovider.InitCloudProvider(s.CloudProvider, s.CloudConfigFile)
+	if cloud != nil {
+		glog.Infof("Successfully initialized cloud provider: %s from the config file: %s\n", s.CloudProvider, s.CloudConfigFile)
+	}
```

Comment:
Arguably we should not Fatal, but as it was :) lets still return the error if in the future it changes. It did give us a return type with an error!